lib/glare.rb

```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```

Comment:
Oh OK I see annotation here to remove freeze I didn't know this

Reply:
:+1: :joy: